templates/web/pages/product_edit/display_input_field.tt.html

```diff
@@ -0,0 +1,27 @@
+<!-- start templates/[% template.name %] -->
+
+<label for="[% field %]">
+    [% fieldtype %]
+    [% IF language.defined %]
+        (<span class="tab_language">[% language %]</span>)
+    [% END %]
+</label>
+
+[% IF field.match('infocard') || field.match('^packaging_text') %]
+    <textarea name="[% field %]" id="[% field %]" lang="${display_lc}">[% value %]</textarea>
```

Comment:
I forgot to change it. Can you please check this `lang="${display_lc}"`

Reply:
It seems to work. It wasn't called for ingredients_text though. I changed it to use the display_field function also for ingredients_text